packages/mask/src/plugins/Trader/SNSAdaptor/trader/ConfirmDialog.tsx

```diff
@@ -128,7 +161,53 @@ export function ConfirmDialogUI(props: ConfirmDialogUIProps) {
     )
     //#endregion
 
-    const staled = !!(executionPrice && !executionPrice.isEqualTo(trade.executionPrice))
+    const staled = !!(executionPrice && !executionPrice.isEqualTo(cacheTrade?.executionPrice ?? 0))
+
+    const isGreatThanSlippageSetting = useGreatThanSlippageSetting(cacheTrade?.priceImpact)
+
+    const alertTip = useMemo(() => {
+        if (currentSlippage >= 150 && currentSlippage < 1000) return null
+
+        return (
+            <Alert className={classes.alert} icon={<InfoIcon className={classes.alertIcon} />} severity="info">
+                {currentSlippage < 150 ? t('plugin_trader_confirm_tips') : t('plugin_trader_price_impact_warning_tips')}
+            </Alert>
+        )
+    }, [currentSlippage])
+
+    const onAccept = useCallback(() => {
+        setPriceUpdated(false)
+        setCacheTrade(trade)
+    }, [trade])
+
+    const onConfirmPriceImpact = useCallback(() => {
+        if (!cacheTrade?.priceImpact) return
+        setTemporarySlippage(new BigNumber(cacheTrade?.priceImpact.multipliedBy(10000).toFixed(0)).toNumber())
```

Comment:
`const UNISWAP_PERCENT_DENOMINATOR = 10000`

Reply:
Replace with `PERCENT_DENOMINATOR` to avoid name ambiguity